Fix pilot 3 mismatches

camillacalmasini opened the pull request:
This PR fixes DCSREAS for issue #18. I updated the spec as follows to match with pilot 3 formatting:
* updated the terms column in codelists for DISCREAS so that the terms are proper case
* updated STUDY TERMINATED BY SPONSOR to Sponsor Decision
* updated WITHDRAWAL BY SUBJECT to Withdrew Consent

Then in the code, I updated to use `create_var_from_codelist`.

bms63 commented:
> @bms63 hi ben! here are the updates i made:
> 
> * ADSL: revert `SITEGR1` to how it was in pilot 3, convert NAs to blanks, drop length attribute
> * ADADAS: convert NAs to blanks and drop length attribute
> * ADAE: convert NAs to blanks
> 
> For ADSL should I also drop the EOSDT variable?

Yes and I think it is just coded in for ADADAS to help calculate another variable. 

Look at all them green check marks!!!

camillacalmasini commented:
@bms63 i made some updates! dropped EOSDT from the adsl spec and re-added it to adtte. for adtte i also changed CNSR from integer to numeric and fixed the formats so they would match pilot 3. LMK if it all looks good :)

bms63 commented:
@camillacalmasini amazing!!
